### PR TITLE
Mixpanel (Export Events) Analytics (Source) initial commit

### PR DESCRIPTION
**Pull request by mohan-h**
Mixpanel (Export Events) Analytics (Source) initial commit

### CODE REVIEW

File: help/sources/tutorials/ui/create/analytics/mixpanel.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+---
+# Create a *Mixpanel* source connection in the UI
+
+This tutorial provides steps for creating a *Mixpanel* source connector using the Platform user interface.
+
+## Overview
+
+[Mixpanel](https://www.mixpanel.com) is a product analytics tool that enables capturing data on how users interact with a digital product. Mixpanel allows you to analyze this product data with simple, interactive reports that let you query and visualize the data with just a few clicks.
+
+This Adobe Experience Platform [sources](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) leverages the [Event Export API > Download](https://developer.mixpanel.com/reference/raw-event-export) to download your event data as it is received and stored within Mixpanel, along with all event properties (including distinct_id) and the exact timestamp the event was fired into Experience Platform.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: - Let's rephrase the first part to: "The Mixpanel [source](link) leverages..."
- Let's put `distinct_id` in backticks in the last sentence

Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Let's rephrase the first part to: "The Mixpanel [source](link) leverages..."

File: help/sources/tutorials/ui/create/analytics/mixpanel.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+Mixpanel uses bearer tokens as an authentication mechanism to communicate with the Mixpanel Event Export API.
+
+## Prerequisites
+
+Before you start configuring the extension you need to have a Mixpanel account. If you do not have one already, go to the Mixpanel [register](https://mixpanel.com/register/) page to register and create your Mixpanel account.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Let's rephrase this to: "The first step in creating a Mixpanel source connection is to ensure that you have a Mixpanel account. If you do not have one already, see the [Mixpanel register](link) page to register and create your account."

Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Let's rephrase this to: "The first step in creating a Mixpanel source connection is to ensure that you have a Mixpanel account. If you do not have one already, see the [Mixpanel register](link) page to register and create your account."

File: help/sources/tutorials/ui/create/analytics/mixpanel.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+### Gather required credentials
+
+In order to connect *Mixpanel* to Platform, you must provide values for the following connection properties:
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| Host | Mixpanel Raw Data Export API endpoint. <br/><br/>Refer to the [Mixpanel API reference](https://developer.mixpanel.com/reference/overview) page, then scroll to the *Raw Data Export API* section if you require any guidance. | *https://data.mixpanel.com*|
+| Username | Service account Username. <br/><br/>Refer to the [Service Accounts](https://developer.mixpanel.com/reference/service-accounts#authenticating-with-a-service-account) page if you require any guidance. | *`xxxxxxxxxxx`.6d4ee7.mp-service-account* |
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Any reason why only `xxxxxxxxxxx` is backticks? If it's means to hide sensitive values, we could maybe just randomize some characters instead of using `xxxxxxxxxxx`

Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Any reason why only `xxxxxxxxxxx` is backticks? If it's means to hide sensitive values, we could maybe just randomize some characters instead of using `xxxxxxxxxxx`

File: help/sources/tutorials/ui/create/analytics/mixpanel.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+| Password | Service account Password. | *8iQbnjuk5.................* |
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: same here, maybe we could just randomize characters for the password since we're just showing format anyway

Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: same here, maybe we could just randomize characters for the password since we're just showing format anyway

File: help/sources/tutorials/ui/create/analytics/mixpanel.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+| projectId | MixPanel Project Id. <br/><br/>Refer to the pages below if you require any guidance. <br/>[Project Settings](https://help.mixpanel.com/hc/en-us/articles/115004490503-Project-Settings) <br/>[Create and Manage Projects](https://help.mixpanel.com/hc/en-us/articles/115004505106-Create-and-Manage-Projects) | *2384945* |
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Small typo, should be a lowercase "p" on Mixpanel

Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Small typo, should be a lowercase "p" on Mixpanel

File: help/sources/tutorials/ui/create/analytics/mixpanel.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+| timezone | Select the timezone to correspond to your project within the Mixpanel account. <br/><br/>Refer to the [Project Settings](https://help.mixpanel.com/hc/en-us/articles/115004490503-Project-Settings) page if you require any guidance.| *Pacific Standard Time* |
+
+The Mixpanel Project Id can be obtained by navigating to the screen below within your account.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Let's rephrase this to: "You can retrieve your Mixpanel Project ID and configure your project's timezone by navigating to the Project Details page of your account in the Mixpanel UI."

Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Let's rephrase this to: "You can retrieve your Mixpanel Project ID and configure your Timezone by navigating to the Project Details page of your account in the Mixpanel UI."

File: help/sources/tutorials/ui/create/analytics/mixpanel.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+![Mixpanel Project Details](../../../../images/tutorials/create/mixpanel-export-events/mixpanel-project-settings.png?lang=en)
+
+The Mixpanel Service Account credentials can be obtained by navigating to the screen below within your account.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Similar rephrasing here: "To retrieve your Mixpanel service account credentials, navigate to the Service Accounts in the Mixpanel UI. 
Also a bit of a nitpick, but can we update this screenshot to one without the "Service Accounts" pop-up bubble appearing with the cursor? Thanks!

Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Similar rephrasing here: "To retrieve your Mixpanel service account credentials, navigate to the Service Accounts in the Mixpanel UI. 
Also a bit of a nitpick, but can we update this screenshot to one without the "Service Accounts" pop-up bubble appearing with the cursor? Thanks!

File: help/sources/tutorials/ui/create/analytics/mixpanel.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+
+![Mixpanel Service Account](../../../../images/tutorials/create/mixpanel-export-events/mixpanel-service-account.png?lang=en)
+
+>[!TIP]
+>
+>* It is suggested to use a service account that doesn't expire.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Is this configurable by a user? Wondering if we need to add additional guidance to this

Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Is this configurable by a user? Wondering if we need to add additional guidance to this

File: help/sources/tutorials/ui/create/analytics/mixpanel.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+Finally, create a Platform [schema](https://experienceleague.adobe.com/docs/experience-platform/xdm/schema/composition.html) required for the Mixpanel Event Export API. Refer also to the [limits](#limits) section further below on this page.
+![Create Schema](../../../../images/tutorials/create/mixpanel-export-events/schema.png?lang=en)
+
+## Connect your *Mixpanel* account
+
+In the Platform UI, select **[!UICONTROL Sources]** from the left navigation bar to access the [!UICONTROL Sources] workspace. The [!UICONTROL Catalog] screen displays a variety of sources with which you can create an account.
+
+You can select the appropriate category from the catalog on the left-hand side of your screen. Alternatively, you can find the specific source you wish to work with using the search option.
+
+Under the *Customer Success* category, select *Mixpanel*, and then select **[!UICONTROL Add data]**.
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Update *Customer Success* to *Analytics*

Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Update *Customer Success* to *Analytics*

File: help/sources/tutorials/ui/create/analytics/mixpanel.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+
+### New account
+
+If you are creating a new account, select **[!UICONTROL New account]**, and then provide a name, an optional description, and your credentials. When finished, select **[!UICONTROL Connect to source]** and then allow some time for the new connection to establish.
+
+![new](../../../../images/tutorials/create/mixpanel-export-events/authentication-new-account.png?lang=en)
+
+Next, add the projectId and select the timezone.
+
+![configuration](../../../../images/tutorials/create/mixpanel-export-events/authentication-configuration.png?lang=en)
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md:77]: Let's hide the IMS Organization on the top right of the screen

Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md:77]: Let's hide the IMS Organization on the top right of the screen

File: help/sources/tutorials/ui/create/analytics/mixpanel.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+   |data.event_name|string|
+   |data.import|boolean|
+   |data.insert_id|string|
+   |data.item_id|string|
+   |data.item_name|string|
+   |data.item_price|string|
+   |data.mp_api_endpoint|string|
+   |data.mp_api_timestamp_ms|integer|
+   |data.mp_processing_time_ms|integer|
+   |data.time|integer|
```
Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Can we wrap all the source fields in backticks here?

Comment (carlojavier) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md]: Can we wrap all the source fields in backticks here?

**Review by carlojavier — CHANGES_REQUESTED** [review]
@mohan-h Thanks for this! review done for UI side of the docs. Will notify you once I finish API side.

File: help/sources/tutorials/api/create/analytics/mixpanel.md [hunk 12]
```diff
@@ -0,0 +1,1072 @@
+---
+keywords: Experience Platform;home;popular topics;sources;connectors;source connectors;sources sdk;sdk;SDK
+solution: Experience Platform
+title: Documentation self-service template
+topic-legacy: tutorial
+description: Learn how to connect Adobe Experience Platform to Mixpanel using the Flow Service API.
+hide: true
+hidefromtoc: true
+---
+# Create a *Mixpanel* source connection in the UI
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/analytics/mixpanel.md]: Title should be "Create a source connection and dataflow for Mixpanel using the Flow Service API"

File: help/sources/tutorials/api/create/analytics/mixpanel.md [hunk 13]
```diff
@@ -0,0 +1,1072 @@
...
+
+This tutorial provides steps for creating a *Mixpanel* source connector using the Platform user interface.
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/analytics/mixpanel.md]: Let's update this to: "The following tutorial walks you through the steps to create a source connection and a dataflow to bring Mixpanel data to Platform using the [Flow Service API](https://developer.adobe.com/experience-platform-apis/references/flow-service/).

File: help/sources/tutorials/api/create/analytics/mixpanel.md [hunk 14]
```diff
@@ -0,0 +1,1072 @@
...
+
+## Overview
+
+[Mixpanel](https://www.mixpanel.com) is a product analytics tool that enables capturing data on how users interact with a digital product. Mixpanel allows you to analyze this product data with simple, interactive reports that let you query and visualize the data with just a few clicks.
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/analytics/mixpanel.md:16]: This high-level overview of the product is nice to have. Thanks for adding!

File: help/sources/tutorials/api/create/analytics/mixpanel.md [hunk 15]
```diff
@@ -0,0 +1,1072 @@
...
+
+This Adobe Experience Platform [sources](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) leverages the [Event Export API > Download](https://developer.mixpanel.com/reference/raw-event-export) to download your event data as it is received and stored within Mixpanel, along with all event properties (including distinct_id) and the exact timestamp the event was fired into Experience Platform.
+
+Mixpanel uses bearer tokens as an authentication mechanism to communicate with the Mixpanel Event Export API.
+
+## Prerequisites
+
+Before you start configuring the extension you need to have a Mixpanel account. If you do not have one already, go to the Mixpanel [register](https://mixpanel.com/register/) page to register and create your Mixpanel account.
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/analytics/mixpanel.md]: Similar change with the UI-side here: "The first step in creating a Mixpanel source connection is to ensure that you have a Mixpanel account. If you do not have one already, see the [Mixpanel register](https://github.com/AdobeDocs/experience-platform.en/pull/64/link) page to register and create your account."

File: help/sources/tutorials/api/create/analytics/mixpanel.md [hunk 16]
```diff
@@ -0,0 +1,1072 @@
...
+---
+# Create a *Mixpanel* source connection in the UI
+
+This tutorial provides steps for creating a *Mixpanel* source connector using the Platform user interface.
+
+## Overview
+
+[Mixpanel](https://www.mixpanel.com) is a product analytics tool that enables capturing data on how users interact with a digital product. Mixpanel allows you to analyze this product data with simple, interactive reports that let you query and visualize the data with just a few clicks.
+
+This Adobe Experience Platform [sources](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) leverages the [Event Export API > Download](https://developer.mixpanel.com/reference/raw-event-export) to download your event data as it is received and stored within Mixpanel, along with all event properties (including distinct_id) and the exact timestamp the event was fired into Experience Platform.
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/analytics/mixpanel.md]: Same revision here: "The Mixpanel [source](https://github.com/AdobeDocs/experience-platform.en/pull/64/link) leverages..."

File: help/sources/tutorials/api/create/analytics/mixpanel.md [hunk 17]
```diff
@@ -0,0 +1,1072 @@
...
+| `fileType=json` | The file type of the file you want to bring to Platform. Currently, `json` is the only supported file type. |
+| `{PREVIEW}` | A boolean value that defines whether the contents of the connection supports preview. |
+| `{SOURCE_PARAMS}` | Defines parameters for the source file you want to bring to Platform. To retrieve the accepted format-type for `{SOURCE_PARAMS}`, you must encode the entire `list_id` string in base64. In the example below, `"list_id": "10c097ca71"` encoded in base64 equates to `eyJsaXN0SWQiOiIxMGMwOTdjYTcxIn0=`. |
+
+
+**Request**
+
+```shell
+curl -X GET \
+    'https://platform.adobe.io/data/foundation/flowservice/connections/70383d02-2777-4be7-a309-9dd6eea1b46d/explore?objectType=rest&object=json&fileType=json&preview=true&sourceParams=eyJsaXN0SWQiOiIxMGMwOTdjYTcxIn0=' \
```
Comment (carlojavier) [inline help/sources/tutorials/api/create/analytics/mixpanel.md]: I think the values we have here for `list_id` and `source_params` are from Mailchimp Members. Can we update these to the Mixpanel ones? Values can just be example, but want to make sure the param for `list_id` is updated to the appropriate verbiage... unless it is also called `list_id` for Mixpanel?

**Review by carlojavier — CHANGES_REQUESTED** [review]
@mohan-h Completed review of API-side. Just a few changes, thanks!

File: help/sources/tutorials/api/create/analytics/mixpanel.md (hunk 13 above)
Comment (mohan-h) [reply help/sources/tutorials/api/create/analytics/mixpanel.md]: Oops sorry CPE.

File: help/sources/tutorials/api/create/analytics/mixpanel.md (hunk 12 above)
Comment (mohan-h) [reply help/sources/tutorials/api/create/analytics/mixpanel.md]: Oops sorry CPE

File: help/sources/tutorials/ui/create/analytics/mixpanel.md (hunk 1 above)
Comment (mohan-h) [reply help/sources/tutorials/ui/create/analytics/mixpanel.md]: The 1st bullet would read odd, have updated _[Event Export API > Download]_ to _[Mixpanel Event Export API > Download]_. The Mixpanel prefix was missed.
2nd bullet, done.

Comment (mohan-h) [reply help/sources/tutorials/ui/create/analytics/mixpanel.md]: The 1st bullet would read odd, have updated [Event Export API > Download] to [Mixpanel Event Export API > Download]. The Mixpanel prefix was missed.

**Comment by mohan-h** [comment]
Committed https://github.com/mohan-h/experience-platform.en/commit/eaae81af7fff3072e27e32d720d960b07177d57a addressing review inputs.

File: help/sources/tutorials/api/create/analytics/mixpanel.md [hunk 18]
```diff
@@ -0,0 +1,1072 @@
...
+    -d '{
+        "name": "{Mixpanel} base connection",
+        "description": "{Mixpanel} base connection to authenticate to Platform",
+        "connectionSpec": {
+            "id": "6360f136-5980-4111-8bdf-15d29eab3b5a",
+            "version": "1.0"
+        },
+        "auth": {
+            "specName": "OAuth2 Refresh Code",
+            "params": {
```
Comment (vivetiwa) [inline help/sources/tutorials/api/create/analytics/mixpanel.md:90]: JSON formatting could be added.

File: help/sources/tutorials/api/create/analytics/mixpanel.md [hunk 19]
```diff
@@ -0,0 +1,1072 @@
...
+
+
+| Parameter | Description |
+| --------- | ----------- |
+| `{BASE_CONNECTION_ID}` | The base connection ID generated in the previous step. |
+| `objectType=rest` | The type of object that you wish to explore. Currently, this value is always set to `rest`. |
+| `{OBJECT}` | This parameter is required only when viewing a specific directory. Its value represents the path of the directory you wish to explore. |
+| `fileType=json` | The file type of the file you want to bring to Platform. Currently, `json` is the only supported file type. |
+| `{PREVIEW}` | A boolean value that defines whether the contents of the connection supports preview. |
+| `{SOURCE_PARAMS}` | Defines parameters for the source file you want to bring to Platform. To retrieve the accepted format-type for `{SOURCE_PARAMS}`, you must encode the entire `{"projectId":"2671127","timezone":"Pacific Standard Time"}` string in base64. In the example below, `"{"projectId":"2671127","timezone":"Pacific Standard Time"}": "10c097ca71"` encoded in base64 equates to `eyJwcm9qZWN0SWQiOiIyNjcxMTI3IiwidGltZXpvbmUiOiJQYWNpZmljIFN0YW5kYXJkIFRpbWUifQ==`. |
```
Comment (vivetiwa) [inline help/sources/tutorials/api/create/analytics/mixpanel.md]: In line `{"projectId":"2671127","timezone":"Pacific Standard Time"}": "10c097ca71"` -`10c097ca71` is not required.

File: help/sources/tutorials/api/create/analytics/mixpanel.md [hunk 20]
```diff
@@ -0,0 +1,1072 @@
...
+```
+
+When performing GET requests to explore your source's file structure and contents, you must include the query parameters that are listed in the table below:
+
+
+| Parameter | Description |
+| --------- | ----------- |
+| `{BASE_CONNECTION_ID}` | The base connection ID generated in the previous step. |
+| `objectType=rest` | The type of object that you wish to explore. Currently, this value is always set to `rest`. |
+| `{OBJECT}` | This parameter is required only when viewing a specific directory. Its value represents the path of the directory you wish to explore. |
```
Comment (vivetiwa) [inline help/sources/tutorials/api/create/analytics/mixpanel.md]: {OBJECT} could be json always in case of SSS sources.

File: help/sources/tutorials/api/create/analytics/mixpanel.md [hunk 21]
```diff
@@ -0,0 +1,1072 @@
...
+    -H 'x-api-key: {API_KEY}' \
+    -H 'x-gw-ims-org-id: {ORG_ID}' \
+    -H 'x-sandbox-name: {SANDBOX_NAME}' \
+    -H 'Content-Type: application/json' \
+    -d '{
+        "name": "{Mixpanel} Source Connection",
+        "description": "{Mixpanel} Source Connection",
+        "baseConnectionId": "70383d02-2777-4be7-a309-9dd6eea1b46d",
+        "connectionSpec": {
+            "id": "6360f136-5980-4111-8bdf-15d29eab3b5a",
```
Comment (vivetiwa) [inline help/sources/tutorials/api/create/analytics/mixpanel.md]: Final connectionSpec id would be `fd2c8ff3-1de0-4f6b-8fa8-4264784870eb`

File: help/sources/tutorials/api/create/analytics/mixpanel.md [hunk 22]
```diff
@@ -0,0 +1,1072 @@
...
+            "version": "1.0"
+        },
+        "data": {
+            "format": "json"
+        },
+        "params": {
+            "projectId": "{MIXPANEL_PROJECT_ID}",
+            "timezone": "{MIXPANEL_PROJECT_ID}"
```
Comment (vivetiwa) [inline help/sources/tutorials/api/create/analytics/mixpanel.md]: typo MIXPANEL_PROJECT_ID should be MIXPANEL_TIMEZONE

File: help/sources/tutorials/api/create/analytics/mixpanel.md [hunk 23]
```diff
@@ -0,0 +1,1072 @@
...
+    }'
+```
+
+| Property | Description |
+| --- | --- |
+| `name` | The name of your source connection. Ensure that the name of your source connection is descriptive as you can use this to look up information on your source connection. |
+| `description` | An optional value that you can include to provide more information on your source connection. |
+| `baseConnectionId` | The base connection ID of *Mixpanel*. This ID was generated in an earlier step. |
+| `connectionSpec.id` | The connection specification ID that corresponds to your source. |
+| `data.format` | The format of the *Mixpanel* data that you want to ingest. Currently, the only supported data format is `json`. |
```
Comment (vivetiwa) [inline help/sources/tutorials/api/create/analytics/mixpanel.md:403]: projectId and timezone parameter missing in table

File: help/sources/tutorials/ui/create/analytics/mixpanel.md [hunk 24]
```diff
@@ -0,0 +1,121 @@
...
+
+The **[!UICONTROL Connect Mixpanel account]** page appears. On this page, you can either use new credentials or existing credentials.
+
+### Existing account
+
+To use an existing account, select the *Mixpanel* account you want to create a new dataflow with, then select **[!UICONTROL Next]** to proceed.
+
+![existing](../../../../images/tutorials/create/mixpanel-export-events/authentication-existing-account.png?lang=en)
+
+### New account
```
Comment (vivetiwa) [inline help/sources/tutorials/ui/create/analytics/mixpanel.md:69]: After new account can we add preview screen with input value.

File: help/sources/tutorials/api/create/analytics/mixpanel.md (hunk 18 above)
Comment (mohan-h) [reply help/sources/tutorials/api/create/analytics/mixpanel.md:90]: Not sure about the json formatting as its a curl request and formatting is prefixed with the ```shell directive. Have added indentation.

File: help/sources/tutorials/api/create/analytics/mixpanel.md (hunk 21 above)
Comment (mohan-h) [reply help/sources/tutorials/api/create/analytics/mixpanel.md]: @vivetiwa There are 4 occurrences of connectionSpec. Changed for all 4 occurrences. 
However within [zendesk api](https://experienceleague.adobe.com/docs/experience-platform/sources/api-tutorials/create/customer-success/zendesk.html?lang=en) only 2 id's match-up. Could you let me know which instances to change.

cc @carlojavier

File: help/sources/tutorials/api/create/analytics/mixpanel.md (hunk 22 above)
Comment (mohan-h) [reply help/sources/tutorials/api/create/analytics/mixpanel.md]: Oops, good spot.

**Comment by mohan-h** [comment]
The documentation for the connector has been made live at:
https://experienceleague.adobe.com/docs/experience-platform/sources/connectors/analytics/mixpanel.html?lang=en
